wpeverest/user-registration
# UR-1035 Compatibility custom captcha

### All Submissions:






### Changes proposed in this Pull Request:



This PR will introduce the Custom Captcha Field. It generates random math equations or predefined question answers on the registration form.

### How to test the changes in this Pull Request:

1. Drag and drop the custom captcha field and verify whether it is working as expected or not on the registration form.


### Types of changes:




### Other information:




### Changelog entry

> UR-1035 Compatibility custom captcha.
